### DISCUSSION TITLE
Can I replace this to hyprpicker to freeze the screen?

### DISCUSSION THREAD

**Discussion by kRHYME7** [Zerodya/wl-freeze#5] [Q&A]
Hi, I needed some guidance on whether the script can replicate the hyprpicker function that freezes the functions on the entire screen. I want to use this to replace the hyprpicker freeze function on grimblast. I would Like to ask if the script is capable of this. Thank You

**Comment by Zerodya** [comment]
Could you better clarify what is your goal? 

I don't use hyprpicker and grimblast so I'm not familiar with their code, but on a quick installation and usage of hyprpicker it looks like it makes a screenshot of the whole screen and puts it a layer above everything else so you can pick the color on the screenshot but the applications continue running in the background.

Meanwhile hyprfreeze stops the applications completely, and there's no way right now to stop the whole screen simply because I don't need it, but I could look into that if there's a real use case.

**Reply by kRHYME7** [reply]
Yups I guess that's how hyprpicker works.

My real issue is totally unrelated to this repo thu, I thought the implementation of pausing  the process is slightly the same with hyprpicker so I can freeze the screen to take a screenshot with combination of grim and slurp(Also grimblast is available which does the same and also same output). The Problem of This set up is I can capture the Cursor of the hyprpicker on the final screenshot. Hyprpicker is compiled and I don't understand how it works in code. So my wild guess as I am always using this hyprfreeze is maybe It has slightly the same implementation  of freezing the screen, If not but maybe doable.

**Reply by Zerodya** [reply]
Your best bet would be opening an issue in either the grimblast or the hyprshot repo and see if they can implement such a feature, or if they know a workaround.

**Reply by kRHYME7** [reply]
Okay, BTW thanks for this awesome utility !!!
